deploy/env-development.json

```diff
@@ -241,7 +241,7 @@
             ]
         },
         {
-            "name": "[concat(take(resourceGroup().name,15), '-sharedKV')]",
+            "name": "[concat(take(resourceGroup().name,10), '-dev-sharedKV')]",
```

Comment:
I think going to 11 would also work here

Reply:
Yep i agree, to be exact, it is 11, but i like round numbers :D i can change it if we want to make it 11